html/browsers/browsing-the-web/unloading-documents/unload/pagehide-manual.html

```diff
@@ -2,4 +2,4 @@
 <title>Document salvagable state after setting pagehide handler</title>
 <script>onpagehide = function() {setTimeout(function(){document.body.innerHTML = "PASS"}, 100)}</script>
 <p>Click the link below then navigate back to this page. Shortly after returning you should see the text "PASS"</p>
-<p><A href="manual-001-1.html">Click here</a>
+<p><a href="pagehide-manual-1.html">Click here</a>
```

Comment:
Clicking will now 404, is that part of the test, or accidental?

Reply:
It's not 404, that file was renamed.

Reply:
Oh, there's both pagehide-manual.html and pagehide-manual-1.html.